### PR TITLE
feat: PubSub Lite Spark Connector

### PR DESCRIPTION
**Pull request by jiangmichaellll**
Moves from java-pubsublite repo to separate repo.

### CODE REVIEW

File: examples/simple_read.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    .readStream \
+    .format('pubsublite') \
+    .option('pubsublite.subscription', full_subscription_path) \
+    .load() \
+    .writeStream \
+    .format('console') \
+    .outputMode('append') \
+    .trigger(processingTime='1 second') \
+    .start() \
+    .awaitTermination()
```
Comment (anguillanneuf) [inline examples/simple_read.py:33]: Can we place this example in `.readme-partials.yaml`?

File: examples/simple_read.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+from pyspark.sql import SparkSession
+import sys
+
+full_subscription_path = sys.argv[1]
+
+spark = SparkSession.builder.appName('Simple PubSub Lite Read').master('yarn').getOrCreate()
+
+spark \
+    .readStream \
```
Comment (anguillanneuf) [inline examples/simple_read.py:24]: nit: missing parenthesis.

File: examples/simple_read.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    .format('pubsublite') \
+    .option('pubsublite.subscription', full_subscription_path) \
+    .load() \
+    .writeStream \
```
Comment (anguillanneuf) [inline examples/simple_read.py:28]: nit: missing parenthesis.

**Review by anguillanneuf — CHANGES_REQUESTED** [review]
Could you mind moving the contents of the current README to `.readme-partials.yaml`?

File: examples/simple_read.py (hunk 2 above)
Comment (jiangmichaellll) [reply examples/simple_read.py:24]: actually adding parenthesis would lead to error as I remember.

File: examples/simple_read.py (hunk 3 above)
Comment (jiangmichaellll) [reply examples/simple_read.py:28]: actually adding parenthesis would lead to error as I remember.

Comment (anguillanneuf) [reply examples/simple_read.py:28]: You are right!

File: examples/simple_read.py (hunk 1 above)
Comment (jiangmichaellll) [reply examples/simple_read.py:33]: https://github.com/googleapis/java-pubsublite-spark/pull/3/commits/a31b488e7d424368282f1b3f475ba3884254ceaa

**Review by anguillanneuf — APPROVED** [review]

**Pull request merged**